packages/desktop/.sentrycliignore

```diff
@@ -1,4 +1,3 @@
-public/
```

Comment:
I don't think we want to ignore `public/` since that's where our build outputs are located